packages/css/src/components/hero-image/hero-image.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-hero-image {
+  aspect-ratio: var(--amsterdam-hero-image-aspect-ratio);
+  display: grid;
+}
+
+.amsterdam-hero-image > * {
+  align-self: center;
+  grid-area: 1 / -1;
+}
+
+.amsterdam-hero-image__image {
+  object-fit: cover;
+  width: 100%;
```

Comment:
This is a theory but if you do not specify `height: 100%` or `overflow: hidden` for the parent and somebody uses a higher image it will not honor the aspect-ratio right?

Reply:
Ok, turns out grid does not handle aspect-ratio, the image will need a wrapper.

Reply:
```suggestion
.amsterdam-hero-image__wrapper {
  aspect-ratio: var(--amsterdam-hero-image-aspect-ratio);
}

.amsterdam-hero-image__image {
  height: 100%;
  object-fit: cover;
  width: 100%;
}
```

Reply:
Ah, good find. I updated for this, slightly differently though.

Reply:
Oh nice, you don't need a wrapper.